main.go

```diff
@@ -320,7 +328,7 @@ func main() {
 			// Gzip output into pipe so that we don't need to store locally
 			reader, writer := io.Pipe()
 			go func(index int) {
-				zippedOutput := gzip.NewWriter(writer) // sorcery
+				zippedOutput, _ := gzip.NewWriterLevel(writer, gzip.BestSpeed) // sorcery
```

Comment:
let's check this error

(or at least leave a comment that we don't need to check it because we're using their constant: https://golang.org/src/compress/gzip/gzip.go?s=1836:1896#L50)

I'd rather that we check it in case someone plays with the value at a later date